egs/librispeech/asr1/conf/tuning/transducer/decode.yaml

```diff
@@ -0,0 +1,6 @@
+batchsize: 0
+beam-size: 60
+ctc-weight: 0
+penalty: 0.0
+search-type: maes #"nsc", "tsd", "alsd", "maes"
```

Comment:
See this [comment](https://github.com/espnet/espnet/pull/4080#discussion_r809960203). Your previous decoding config is equivalent to the one I shared (with `beamsize` decreased).

If you want to use mAES instead, you will need to tune the algorithm parameters accordingly (see the [documentation](https://espnet.github.io/espnet/tutorial.html#inference)).

Reply:
Okay. Thanks!